### PR TITLE
docs: updated changelog with features in 2.05.0.1.0

### PR DESCRIPTION
**Pull request by pavitthrap**
Updated changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,46 @@ All notable changes to this project will be documented in this file.
...
+  `BlocksAvailable` and `MicroblocksAvailable` messages for block and microblock 
+  streams that this node does not have. If such messages arrive during an inventory 
+  sync, the p2p state machine will immediately transition from the inventory sync 
+  work state to the block downloader work state, and immediately proceed to fetch 
+  the available block or microblock stream.
+
+### Fixed 
+- Updates the lookup key for contracts in the pessimistic cost estimator. Before, contracts
+  published by different principals with the same name would have had the same 
+  key in the cost estimator.
```
Comment (kantai) [inline CHANGELOG.md:47]: This is the same update as the one described at line 29, but it's better described here, so you can delete the entry at line 29.

**Review by kantai — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pavitthrap) [reply CHANGELOG.md:47]: oh yup good catch.

**Review by gregorycoppola — COMMENTED** [review]
@pavitthrap can you sync to pick up the description of the new fee estimator :) ?

**Comment by gregorycoppola** [comment]
need to push

File: README.md [hunk 2]
```diff
@@ -402,8 +402,8 @@ discussed in [Versioning](#versioning).  We assume, in this section,
 that the change is not consensus-breaking.  So, the release manager must first
 determine whether there are any "non-consensus-breaking changes that require a
 fresh chainstate". This means, in other words, that the database schema has
-changed. Then, the release manager should determine whether this is a feature
-release, as opposed to a hot fix or a patch. Given the answers to these
+changed without a migration. Then, the release manager should determine whether this
```
Comment (gregorycoppola) [inline README.md:405]: ", but an automatic migration is no implemented."

**Review by gregorycoppola — APPROVED** [review]
One suggestion to consider.

**Pull request merged**